docs/source/Makefile

```diff
@@ -190,3 +191,6 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+serve: html
```

Comment:
i don't think `http.server` actually takes a `--directory` argument! (only confirmed on python 3.6.6)

Reply:
Hmm. Maybe they introduced it on python 3.7+. I will swap that with a simple `cd`.